server/src/server/api/TopicsInterface.ts

```diff
@@ -12,13 +14,29 @@ export class TopicsInterface {
 
         app.post('/topics', async function (req: Request, res: Response) {
             console.log('post /topics');
-
-            await topic.createTopic(
-                req.body.name,
-                parseInt(req.body.partitionCount),
-            );
-
-            res.sendStatus(200);
+            let good : boolean = true;
+
+            try {
+                await topic.createTopic(
+                    req.body.name,
+                    parseInt(req.body.partitionCount),
+                );
```

Comment:
if you put the ` res.sendStatus(200);` here then you can throw away the good variable